### DISCUSSION TITLE
A command handler with no response value bypasses the validation behavior (MediatR + FluentValidation)

### DISCUSSION THREAD

**Discussion by safronov-dmitrii** [LuckyPennySoftware/MediatR#878] [Q&A]
Hello

After updating the MediatR library to 12.0.1, I ran into a problem.

A command handler with no response value inherited from the IRequestHandler interface bypasses the validation behavior (IPipelineBehavior).

You can see the problem in the [demo project](https://github.com/safronov-dmitrii/Demo), note that the validation doesn't work. 
However, in version 11.1.0, where the return value of Task<Unit.> was provided in the command handler for, everything works.

**Comment by jbogard** [comment]
See the upgrade guide, mainly the section on behaviors:

https://github.com/jbogard/MediatR/wiki/Migration-Guide-11.x-to-12.0#rolling-back-generic-constraints

It looks like your behavior was not corrected.

**Reply by safronov-dmitrii** [reply]
Thanks a lot! Your answer helped me.
